docs/remote-control/operating-the-remote.md

```diff
@@ -53,23 +68,22 @@ button, for example:
 The fixture screen allows you to select fixtures exactly as you would on
 the console.
 
-![Fixture View in Titan Remote Android App](/docs/images/Fixture-View-in-Titan-Remote-Android-App.png)
-
-You can then use the **Palette** screens to control the fixtures, or press
-the **IPCGBES** buttons to directly control functions of the fixtures using
-wheels.
+![Fixture View in Titan Remote Android App](/docs/images/Remote-Fixtures.png)
 
 You can scroll the fixture screen up and down by dragging the window. To
-view the fixture pages, drag the Pages bar to the right.
+view the fixture pages, drag the title bar of the fixture window downwards.
 
 Use the **Highlight**, **Prev** and **Next** buttons to step through a set of selected fixtures
 one at a time. Press the **All** button to reselect all fixtures in the set. Hold down the **All** button to
 step through various selection patterns (odd, even etc).
 
-![Fixture Control in Titan Remote Android App](/docs/images/Fixture-Control-in-Titan-Remote-Android-App.png)
+Use the **IPCGBES** buttons at the top to select the group of attributes you want
+to control, and scroll wheels will be shown for the available functions.
+
+![Colour Control in Titan Remote Android App](/docs/images/Remote-Colour-Attributes.png)
 
 The wheels show function values (such as colours or gobo names). To go
-to the next function, just touch the arrows at the top or bottom of the
+to the next function if the attribute has ranges, just touch the arrows at the top or bottom of the
```

Comment:
If this is the only section on the wheels we should probably mention that it is possible to pinch to fan an attribute.

We should probably mention that for function ranges you need to tap up/down.